api_docs/deprecations.mdx

```diff
@@ -126,23 +124,25 @@ warning: This document is auto-generated and is meant to be viewed inside our ex
 | <DocLink id="kibDataPluginApi" section="def-public.IIndexPattern" text="IIndexPattern"/> | [types.ts#L16](https://github.com/elastic/kibana/tree/master/src/plugins/discover/public/application/embeddable/types.ts#L16) | - |
 | <DocLink id="kibDataPluginApi" section="def-public.IIndexPattern" text="IIndexPattern"/> | [types.ts#L30](https://github.com/elastic/kibana/tree/master/src/plugins/discover/public/application/embeddable/types.ts#L30) | - |
 | <DocLink id="kibDataPluginApi" section="def-public.IFieldType" text="IFieldType"/> | [kibana_services.ts#L104](https://github.com/elastic/kibana/tree/master/src/plugins/discover/public/kibana_services.ts#L104) | - |
-| <DocLink id="kibDataPluginApi" section="def-public.IFieldType" text="IFieldType"/> | [search_embeddable.ts#L23](https://github.com/elastic/kibana/tree/master/src/plugins/discover/public/application/embeddable/search_embeddable.ts#L23) | - |
```

Comment:
@weltenwort As far as I understand these changes are done automatically reading changes and comments in the source code, right? To understand a lit bit better the documentation system, did you have to do any manual changes in this deprecations.mdx file?

Reply:
These changes were made by the `scripts/build_api_docs` script, which I ran locally to satisfy jenkins. It's part of the new experimental documentation system, but I don't know what these changes mean. :shrug: